lib/travis/hub/config.rb

```diff
@@ -40,7 +40,7 @@ def jwt_key(type)
              limit:          { resets: { max: 50, after: 6 * 60 * 60 } },
              notifications:  [ 'billing' ],
              auth:           { jwt_private_key: jwt_key(:private), jwt_public_key: jwt_key(:public), http_basic_auth: http_basic_auth },
-             billing:        { url: ENV['BILLING_URL'] || 'http://localhost:9292', auth: ENV['BILLING_AUTH'] || 't0Ps3Cr3t' }
+             billing:        { url: ENV['BILLING_URL'] || 'http://localhost:9292', auth_key: ENV['BILLING_AUTH_KEY'] || 't0Ps3Cr3t' }
```

Comment:
@piotrmilcarz thank you, just curious to know why hardcode the `auth` value to `t0Ps3Cr3t`

Reply:
@shairyar I'm guessing, it's just a mock for tests, read it as "topsecret" :)

Reply:
It was used just for testing purposes. I hope it's not one of used passwords in real world ;-)